pkg/sidecar/config.go

```diff
@@ -215,6 +217,9 @@ func NewAgentLimit() AgentLimit {
 // AgentBreaker is a map of destination names to their associated circuit breaker specs.
 type AgentBreaker map[string]specs.CircuitBreakerSpec
 
+// AgentRetryTimeout is a map of destination names to their associated retry/timeout specs.
+type AgentRetryTimeout map[string]v1alpha1.RetryTimeoutConfigSpec
```

Comment:
This probably isn't needed after some cleanup on the controller side. Especially since the agent won't use it.